setup.py

```diff
@@ -119,7 +119,7 @@ def get_git_sha():
         'requests-kerberos==0.12.0',
         'django==2.1.5',
         'wtforms-json',
-        'pyhive>=0.6.1',
+        'pyhive<=0.6.1',
```

Comment:
why we have put `<` instead?

Reply:
so that it gets aligned with version mentioned in  Pipfile

Reply:
As discussed, please revert to original, i.e. `pyhive>=0.6.1`